tasks/deps-debian.yml

```diff
@@ -14,6 +14,6 @@
     src: /usr/lib/x86_64-linux-gnu/liblua5.3-posix.so
     dest: /usr/lib/x86_64-linux-gnu/lua/5.3/posix.so
     state: link
-  when: ansible_distribution_release == 'bionic'
+  when: ansible_distribution_release == 'bionic' and ansible_distribution_release == 'focal'
```

Comment:
This can't be true anymore, because `ansible_distribution_release` can't be **bionic** and **focal** at the same time. What I meant in my previous review was to check **both** distribution and distribution release.

Reply:
Thanks I think I misread your previous comments. Please find the correct version in the most recent commit.